job-server/config/shiro.ini

```diff
@@ -0,0 +1,16 @@
+
+# use this for basic ldap authorization, without group checking
+activeDirectoryRealm = org.apache.shiro.realm.ldap.JndiLdapRealm

```

Comment:
I'm wondering if this should be named `shiro.ini.template`, and adding to the deploy section of the README, because people could deploy with this and not realize they have to change the settings.